Sources/Testing/Support/CError.swift

```diff
@@ -27,8 +27,12 @@ struct CError: Error, RawRepresentable {
 /// [here](https://learn.microsoft.com/en-us/windows/win32/debug/system-error-codes).
 ///
 /// This type is not part of the public interface of the testing library.
-struct Win32Error: Error, RawRepresentable {
-  var rawValue: DWORD
+package struct Win32Error: Error, RawRepresentable {
+  package var rawValue: CUnsignedInt
+
+  package init(rawValue: CUnsignedInt) {
```

Comment:
Why change this to `CUnsignedInt`? Win32 errors are generally `DWORD`. They are size compatible of course.

Reply:
If we link to the WinSDK module, you can't write tests for that module's overlay. So our definition of the type comes from our _TestingInternals module instead, and its symbols/types can't be visible in the Swift interface. So we substitute the currency type instead.